database/seeders/DevelopmentSeeder.php

```
@@ -35,131 +36,347 @@ final class DevelopmentSeeder extends Seeder
+                return $skills;
+            }
+        }
+
+        return ['Git', 'Agile'];
+    }
+
+    private function createRealisticApplications(Collection $candidates, JobRequisition $requisition): void
+    {
+        $stages = $requisition->stages->sortBy('display_order');
+
+        $candidates->each(function ($candidate, $index) use ($requisition, $stages): void {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$index'. (undefined)

(UnusedFormalParameter)